.github/workflows/push-tags.yaml

```
@@ -0,0 +1,28 @@
+name: Push Tag
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release version (e.g. v0.1.0)'
```

nit: use a recent version for the example
```suggestion
        description: 'Release version (e.g. v1.7.0)'
```